### PR TITLE
Fix nullable?

### PR DESCRIPTION
**Pull request by rocktavious**
Resolves https://github.com/OpsLevel/team-platform/issues/452

### Problem

Unable to set the `value` field to `null`

### Solution

Handle null value properly

### Checklist

- [ ] I have run this code, and it appears to resolve the stated issue.
- [ ] This PR does not reduce total test coverage
- [ ] This PR has no user interface changes or has already received approval from product management to change the interface.
- [ ] Make a [changie](https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/CONTRIBUTING.md#changie-change-log-generation) entry that explains the customer facing outcome of this change

### CODE REVIEW

File: opslevel/resource_opslevel_property_assignment.go [hunk 1]
```diff
@@ -106,11 +106,12 @@ func (resource *PropertyAssignmentResource) Create(ctx context.Context, req reso
...
 	definition := planModel.Definition.ValueString()
 	owner := planModel.Owner.ValueString()
-	value := opslevel.JsonString(planModel.Value.ValueString())
 	input := opslevel.PropertyInput{
 		Definition: *opslevel.NewIdentifier(planModel.Definition.ValueString()),
 		Owner:      *opslevel.NewIdentifier(planModel.Owner.ValueString()),
-		Value:      value,
+	}
+	if planModel.Value.IsNull() {
+		input.Value = opslevel.JsonString(planModel.Value.ValueString())
```
Comment (davidbloss) [inline opslevel/resource_opslevel_property_assignment.go:114]:
```suggestion
		input.Value = opslevel.JsonString("")
```
Totally optional, but would this be clearer?

Comment (rocktavious) [reply opslevel/resource_opslevel_property_assignment.go:114]: 🤦 the null check is suppose to be "no null". 🤦

File: opslevel/resource_opslevel_property_assignment.go [hunk 2]
```diff
@@ -106,11 +106,12 @@ func (resource *PropertyAssignmentResource) Create(ctx context.Context, req reso
 
 	definition := planModel.Definition.ValueString()
 	owner := planModel.Owner.ValueString()
-	value := opslevel.JsonString(planModel.Value.ValueString())
 	input := opslevel.PropertyInput{
 		Definition: *opslevel.NewIdentifier(planModel.Definition.ValueString()),
 		Owner:      *opslevel.NewIdentifier(planModel.Owner.ValueString()),
-		Value:      value,
+	}
+	if planModel.Value.IsNull() {
```
Comment (rocktavious) [inline opslevel/resource_opslevel_property_assignment.go:113]:
```suggestion
	if !planModel.Value.IsNull() {
```

**Review by davidbloss — APPROVED** [review]
LGTM 🎉

**Pull request merged**